source/node/introduction.rst

```diff
@@ -208,295 +215,414 @@ When using ropsten testnet, follow the steps mentioned in this
+a. Alice sends a payment channel opening request to Bob.
 
-   a. [Alice] Send a request to open a channel with bob:
+.. code-block::
 
-   .. code-block::
+   > # [In Alice's CLI]
+   > channel send-opening-request bob 1 2
+
+b. Bob receives a notification in his CLI. The incoming request contains a request ID.
 
-      # [Alice]
-      channel send-opening-request bob 1 2
+.. code-block::
 
-   b. [Bob] Receives a channel opening request notification that includes
-   request ID. Type the command to accept the channel opening request directly
-   after receiving the notification:
+   > # [In Bob's CLI]
+   > Channel opening request notification received. Notification Alias: request_1_alice.
+   Currency: ETH, Balance: [alice:1.000000 self:1.000000].
+   Expires in 10s.
 
-   .. code-block::
+.. note::
+   Request ID is an identifier assigned by perun-node cli application to
+   reference the request when accepting/rejecting it.
 
-      # [Bob]
-      channel accept-opening-request request_1_alice
+c. Bob accepts the request before it expires.
 
-Once successfully accepted, information on the opened channel is printed in
-both terminals.
+.. code-block::
 
-5. Listing out open channels. In any of the terminals, type the below command
-   to see the list of open channels:
+   > # [In Bob's CLI]
+   > channel accept-opening-request request_1_alice
+
+d. Once Bob accepts the request, the channel will be funded on-chain. Once it
+   is opened, both Alice and Bob receive notifications in their CLIs.
 
 .. code-block::
 
-   channel list-open-channels
+   > # [In Alice's CLI]
+   Channel opened. Alias: ch_1_bob.
+   ID: 94241c75d058186f40826be7ae0803f7731a423903c494faa05b347443bb0a4f, Currency: ETH, Version: 0, Balance [bob:1.000000 self:1.000000].
 
-6. Sending a request to open a payment channel and rejecting it.
+   Subscribed to payment notifications on channel ch_1_bob (ID: 94241c75d058186f40826be7ae0803f7731a423903c494faa05b347443bb0a4f)
 
-   a. [Bob] Send a request to open a channel with bob:
+.. code-block::
 
-   .. code-block::
+   > # [In Bob's CLI]
+   Channel opened. Alias: ch_1_alice.
+   ID: 94241c75d058186f40826be7ae0803f7731a423903c494faa05b347443bb0a4f, Currency: ETH, Version: 0, Balance [alice:1.000000 self:1.000000]
 
-      # [Bob]
-      channel send-opening-request alice 3 4
+   Subscribed to payment notifications on channel ch_1_bob (ID: 94241c75d058186f40826be7ae0803f7731a423903c494faa05b347443bb0a4f)
 
-   b. [Alice] Receives a channel opening request notification that includes
-   request ID. Reject it:
 
-   .. code-block::
+.. note::
+
+  Channel alias is an identifier assigned by perun-node cli application to
+  reference the channel when sending/receiving payments or closing it. It will
+  be different for alice and bob, as it local to the perun-node cli instance.
 
-      # [Alice]
-      channel reject-opening-request request_1_bob
+  In this case, it is ``ch_1_bob`` for ``Alice`` and ``ch_1_alice`` for ``Bob``.
```

Comment:
Indentation is not consistent. in other note blocks its 3 chars...

Reply:
Using node config file - node.yaml
Running perun node with the below config:
{LogLevel:             "debug",
 LogFile:              "",
 ChainURL:             "ws://127.0.0.1:8545",
 Adjudicator:          "0x9daEdAcb21dce86Af8604Ba1A1D7F9BFE55ddd63",
 Asset:                "0x5992089d61cE79B6CF90506F70DD42B8E42FB21d",
 ChainConnTimeout:     10s,
 OnChainTxTimeout:     1m0s,
 ResponseTimeout:      10s,
 CommTypes:            ["tcp"],
 IDProviderTypes:      ["local"],
 CurrencyInterpreters: ["ETH"]}.